Sources/PrivacyDashboard/BrokenSiteReporting/BrokenSiteReport.swift

```diff
@@ -220,7 +220,8 @@ public struct BrokenSiteReport {
             "reportFlow": reportFlow.rawValue,
             "openerContext": openerContext?.rawValue ?? "",
             "vpnOn": vpnOn.description,
-            "userRefreshCount": String(userRefreshCount)
+            "userRefreshCount": String(userRefreshCount),
+            "locale": Locale.autoupdatingCurrent.identifier
```

Comment:
could you perhaps add this as a property 
`let localeIdentifier`
and to the init with a default argument like this:
`localeIdentifier = Locale.current.identifier`
so it is easier to mock it in the tests, and wouldn't rely on system preferences?